armonik/variables.tf

```
@@ -164,8 +164,50 @@ variable "admin_gui" {
   default = null
 }
 
-# Parameters of old admin gui
-variable "admin_old_gui" {
+# Deprecated, must be removed in a future version
+# Parameters of admin gui v0.8 (previously called old admin gui)
+variable "admin_0_8_gui" {
+  description = "Parameters of the old admin GUI"
+  type = object({
+    api = object({
+      name  = string
+      image = string
+      tag   = string
+      port  = number
+      limits = object({
+        cpu    = string
+        memory = string
+      })
+      requests = object({
+        cpu    = string
+        memory = string
+      })
+    })
+    old = object({
```

I know that is was already there, but I think this parameter should be renamed to `app`. We are already in a `admin_0_8_gui` parameter.

```suggestion
    app = object({
```